packages/fxa-content-server/server/lib/amplitude.js

```diff
@@ -101,6 +101,30 @@ const EVENTS = {
     group: GROUPS.rp,
     event: 'engage',
   },
+  'flow.email-domain-validation.engage': {
+    group: GROUPS.registration,
+    event: 'domain_validation_engage',
```

Comment:
@irrationalagent, @davismtl - do we want these reported as individual events, or as event properties on the submit event?